WTWSMS/common/traits/WTWSMS_SPARTA.txt

```diff
@@ -0,0 +1,10 @@
+lame_spartan = {
+	potential = {
+		culture = spartan
+	}
+
+	martial = -6
+	diplomacy = 2
+	stewardship = 2
+	learning = 2
```

Comment:
They reflect Spartan society's direction of the cripples to other disciplines, which they mastered.